ARCHITECTURE.md

```diff
@@ -11,20 +11,18 @@ This app consists of
 
 ## App startup
 
-At startup this app will initialise the db table and setup the webhook channel with Pelion Device management. The app will start a request for subscriptions to device resources.
+At startup the app will initialise the db table and setup a [callback channel (webhook)](https://www.pelion.com/docs/device-management/current/integrate-web-app/event-notification.html) with Pelion Device management. The app will then send a request to initiate [subscriptions to device resources](https://www.pelion.com/docs/device-management/current/connecting/resource-change-webapp.html).
 
 ![](docs/app_setup.png)
 
 ## Device resource changes
 
-When the resource value on a device changes, the notification channel is updated with this information and the web app stores each notification in the postgres database
+When the resource value on a device changes, the notification channel is updated with this information and the web app stores each notification in the postgres database.
 
 ![](docs/resource_notify.png)
 
 ## Visualising Data
 
-The included react app requests all data from the server and visualises this using the recharts library
+The included react app requests all data from the server and visualises this using the [Recharts](http://recharts.org/en-US/) library.
 
-![](docs/view_data.png)
-
-![](docs/app.png)
+![](https://i.ibb.co/8bwXbLS/web-app-demo.gif)
```

Comment:
This gif doesn't seem to be animated.  What is it replacing?

Reply:
it replaces the [static image](https://github.com/bertfroeba/workshop-web-app/blob/master/docs/app.png) with the animated gif.